apps/scheduler/internal/timers/watched.go

```diff
@@ -5,21 +5,25 @@ import (
 	"time"
 
 	"github.com/samber/lo"
-	"github.com/satont/twir/apps/scheduler/internal/types"
+	"go.uber.org/zap"
+
 	model "github.com/satont/twir/libs/gomodels"
 	"github.com/satont/twir/libs/grpc/generated/watched"
-	"go.uber.org/zap"
+
+	"github.com/satont/twir/apps/scheduler/internal/types"
 )
 
 func NewWatched(ctx context.Context, services *types.Services) {
-	timeTick := lo.If(services.Config.AppEnv != "production", 15*time.Second).Else(5 * time.Minute)
+	timeTick := lo.If(services.Config.AppEnv != "production", 3*time.Second).Else(5 * time.Minute)
```

Comment:
Back it to 15 please if you was testing it.

Reply:
fixed